### PR TITLE
Added doc for Emacs + Cider to GettingStarted.adoc below the doc for IntelliJ

### PR DESCRIPTION
**Pull request by markusalbertgraf**
I tried to follow the formatting.

### CODE REVIEW

**Comment by claudiuapetrei** [comment]
@markusalbertgraf there's a typo `src/def/user.clj`

File: GettingStarted.adoc [hunk 1]
```diff
@@ -186,6 +186,23 @@ Within IntelliJ:
...
+----
+  :profiles {:dev {:source-paths ["src/dev" "src/main"]
+                   :repl-options {:nrepl-middleware [cemerick.piggieback/wrap-cljs-repl]}
+                   :dependencies [[binaryage/devtools "0.9.4"]
+                                  [com.cemerick/piggieback "0.2.1"]
+                                  [org.clojure/tools.namespace "0.3.0-alpha4"]
+                                  [figwheel-sidecar "0.5.11"]
+                                  [org.clojure/tools.nrepl "0.2.13"]]}})
+----
+* With `src/def/user.clj` open in a buffer, choose `M-x cider-jack-in-clojurescript` or `C-c M-J` (`M-J` with capital J).
```
Comment (awkay) [inline GettingStarted.adoc]: dev instead of def

Comment (markusalbertgraf) [inline GettingStarted.adoc]: Thank you! I fixed that.

**Comment by awkay** [comment]
thanks!

**Pull request merged**